### PR TITLE
use number_field instead of text_field to improve usability

### LINKED ISSUE #1249: Design improvement: Use number inputs for numbers in patient_information partial
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

@eheintzelman has pointed out that we should use number fields for numbers, and she's def right!

* **What feature or behavior is this required for?**

Clean data!

* **How could we solve this issue? (Not knowing is okay!)**

In the patient information partial (`app/views/patients/_patient_information.html.erb`) we should change text fields to numbers. Specifically, we should do this for:

* Age
* Minors in household
* Adults in household

I THINK this is as simple as changing `text_field` to `number_field` -- check the docs of our form library here: https://github.com/bootstrap-ruby/rails-bootstrap-forms/tree/legacy-2.7 

* **Anything else?**

This started as something else. See below.

<strike>
* **What are we trying to do?**

@eheintzelman has very astutely pointed out that it's kinda clunky to us a toggle-spinner-deal for dropdowns (per below), and that a downward facing chevron would be better here.

* **What feature or behavior is this required for?**

A clean design where it's clear what certain inputs are looking for, datatype wise.

* **How could we solve this issue? (Not knowing is okay!)**

Conveniently, I have no idea! I'd like to avoid some jquery-ui plugin if possible but am not opposed if that's the path of least resistance.

* **Anything else?**

When we settle on a styling, we'll want to apply this in particular to input boxes on patient information, abortion information, and pledge fulfillment tabs, as well as the create patient form.

See this comment for origins. https://github.com/DCAFEngineering/dcaf_case_management/issues/1242#issuecomment-328695235

I'm tagging as UX/question because I don't have a terribly clear strategy/path to victory on this one, and would like people's opinions.
</strike>

### PR DESCRIPTION
**Pull request by mdworken**
Makes text forms for age and any $ amount (eg procedure cost) use number_field instead of text_field

* Fixes #1249

### CODE REVIEW

**Comment by mdworken** [comment]
No longer a WIP. Could someone review please?

**Comment by lomky** [comment]
Screen cap of the cost fields. Age looks similar.

number fields

**Comment by lomky** [comment]
@colinxfleming for final :+1: and :shipit:

File: app/views/patients/_patient_information.html.erb [hunk 1]
```diff
@@ -113,4 +113,4 @@
       </div>
     </div>
   <% end %>
-</section>
+</section>
```
Comment (colinxfleming) [inline app/views/patients/_patient_information.html.erb]: Not a blocker here, but generally you want to have newlines at the end of files. This is the source of a lot of linux arguing/opinions, but it is kind of a good practice! Here's some of said arguing: https://unix.stackexchange.com/questions/23903/should-i-end-my-text-script-files-with-a-newline

**Comment by colinxfleming** [comment]
Looking good, thanks a ton @mdworken !

**Pull request merged**